packages/app/src/components/nominationInfo/index.js

```diff
@@ -9,8 +9,19 @@ function NominationInfo() {
     ActiveNominationContext
   );
 
-  const diffDays = Math.round(Math.abs((activeNomination.admissionDate
-     - activeNomination.dischargeDate) / (24*60*60*1000))) >= 21 ? 'Yes' : 'No';  /* <- hours*minutes*seconds*milliseconds */
+  const { admissionDate } = activeNomination;
+
+  const year = admissionDate !== undefined ?  (admissionDate.toString().split("T")[0].split("-")[0]) : null;
```

Comment:
I like the creativity with the string splitting!  One question, is it necessary to perform this operation 3 separate times? 

What are your thoughts on setting a variable equal to admissionDate.toString().split("T")[0].split("-")?  For example:

`const dateArr = admissionDate ? admissionDate.toString().split("T")[0].split("-") : []`

Then, we could set properDateFormat equal to '${dateArr[0]}/${dateArr[1]}/${dateArr[2]}'

Reply:
Okay, awesome! I updated the branch to reflect these changes. Thanks for the refactoring tip!